index.html

```diff
@@ -5083,29 +5081,28 @@ <h3>Status</h3>
 
         <p>
 If the <code>credentialStatus</code> property is available, the status of a
-<a>verifiable credential</a> is expected to be evaluated by the
-<a>verifier</a> according to the <code>credentialStatus</code> <a>type</a>
-definition for the <a>verifiable credential</a> and the <a>verifier's</a>
-own status evaluation criteria. For example, a <a>verifier</a> can ensure that
-the status of the <a>verifiable credential</a> is not "withdrawn for cause
-by the <a>issuer</a>".
+<a>verifiable credential</a> is expected to be evaluated by the <a>verifier</a>
+according to the <code>credentialStatus</code> <a>type</a> definition for the
+<a>verifiable credential</a> and the <a>verifier's</a> own status evaluation
+criteria. For example, a <a>verifier</a> can ensure the status of the
+<a>verifiable credential</a> is not "withdrawn for cause by the <a>issuer</a>".
         </p>
       </section>
 
       <section class="informative">
         <h3>Fitness for Purpose</h3>
 
         <p>
-The custom <a>properties</a> in the <a>verifiable credential</a> are appropriate
-for the <a>verifier's</a> purpose. For example if a <a>verifier</a> needs to
-determine whether a <a>subject</a> is older than 21 years of age, they might
-rely on a specific <code>birthdate</code> <a>property</a>, or more abstract
-<a>properties</a>, such as <code>ageOver</code>.
+Whether the custom <a>properties</a> in the <a>verifiable credential</a> are
```

Comment:
This does not seem to be a complete sentence.
```
Whether the custom <a>properties</a> in the <a>verifiable credential</a> are
appropriate for the <a>verifier's</a> purpose.
```

Reply:
How about "Fitness for purpose is about whether the custom properties... "?

Reply:
Works for me

Reply:
Updated.